index.js

```diff
@@ -1,11 +1,134 @@
 const core = require('@actions/core');
+const github = require('@actions/github')
+const semver = require('semver');
+const { getConfig } = require('./config');
+const { extractPRNumber, fetchPR, getReleaseType, getReleaseNotes } = require("./pr");
+const { octokit } = require("./octokit");
 
 async function run() {
     try {
-        core.info(`Hello, World!`);
-    } catch (error) {
-        core.setFailed(error.message);
+        var config = getConfig()
```

Comment:
i believe var is old scool and `let` is now for scoped variables

Reply:
Yep. Been a while since I've written JS.